examples/code-runner/code_runner_agent.py

```diff
@@ -103,25 +97,19 @@ async def docs_retriever(url: str) -> str:
         RecursiveUrlLoader,
     )
 
-    loader = RecursiveUrlLoader(
-        url=url, max_depth=20, extractor=lambda x: BeautifulSoup(x, "html.parser").text
-    )
+    loader = RecursiveUrlLoader(url=url, max_depth=20, extractor=lambda x: BeautifulSoup(x, "html.parser").text)
     docs = loader.load()
 
     # Sort the list based on the URLs and get the text
     d_sorted = sorted(docs, key=lambda x: x.metadata["source"])
     d_reversed = list(reversed(d_sorted))
 
-    concatenated_content = "\n\n\n --- \n\n\n".join(
-        [doc.page_content for doc in d_reversed]
-    )
+    concatenated_content = "\n\n\n --- \n\n\n".join([doc.page_content for doc in d_reversed])
     return concatenated_content
 
 
 @env.task
-async def generate(
-    question: str, state: AgentState, concatenated_content: str, debug: bool
-) -> AgentState:
+async def generate(question: str, state: AgentState, concatenated_content: str, debug: bool) -> AgentState:
```

Comment:
you are using the wrong version of `ruff`

Reply:
i'm using ruff 0.12.7. which version are you using